tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "include": [
+    "./src/**/*",
+    "index.d.ts"
+  ],
+  "exclude": [
+    "node_modules"
+  ],
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "lib": [
+      "dom",
+      "es2017"
+    ],
+    "jsx": "react",
+    "strict": true,
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "noEmit": true,
+    "moduleResolution": "node",
+    "skipLibCheck": true
+  }
+}
```

Comment:
Basically copied from [the Gatsby with Typescript example](https://github.com/gatsbyjs/gatsby/blob/master/examples/using-typescript/tsconfig.json), need to use the added `index.d.ts` file.